cypress/components/toast/toast.cy.tsx

```diff
@@ -204,6 +207,19 @@ context("Testing Toast component", () => {
           .should("have.css", "justify-content", align);
       }
     );
+    it.each(["top", "bottom"] as ToastProps["alignY"][])(
+      "should render Toast component alignY prop set to %s",
+      (alignY) => {
+        CypressMountWithProviders(<ToastComponent alignY={alignY} />);
+
+        toastComponent()
+          .parent()
+          .parent()
+          .parent()
+          .parent()
```

Comment:
**thought**: it might be worth when we migrate these tests to Playwright to amend how we are locating the portal element here.

It's not essential now, but if we can directly locate the portal element instead of traversing the DOM, this should make the test less fragile if we ever change the component's markup.